src/components/core/TextArea/TextArea.tsx

```diff
@@ -125,6 +125,7 @@ export function AutoResizingTextAreaWithCharCount({
 
   const handleChange = useCallback(
     (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+      console.log('handling');
```

Comment:
left this in here to make it easier to validate the behavior. can remove before merging